### PR TITLE
[ENH] add `hemi` entity to schema

### LINKED ISSUE #916: add `hemi` entity to the schema
I think it is one of the derivative entities missing from the schema.

https://bids-specification.readthedocs.io/en/stable/05-derivatives/03-imaging.html#discrete-surface-segmentations

Should we add it?

### PR DESCRIPTION
**Pull request by Remi-Gau**
fixes #916 

it had slipped through the cracks

### CODE REVIEW

File: src/schema/objects/entities.yaml [hunk 1]
```diff
@@ -91,6 +91,16 @@ flip:
     value which needs to be stored in the field `FlipAngle` of the separate JSON file.
   type: string
   format: index
+hemisphere:
+  name: Hemisphere
+  entity: hemi
+  description: |
+    The hemi entity is used for storing information about a structure that is restricted to a hemibrain.
```
[the remark this answers is not in the document]
Comment (Remi-Gau) [reply src/schema/objects/entities.yaml:98]: Good with me.

**Comment by Remi-Gau** [comment]
> a link to the entity appendix section in the spec text where `hemi` is mentioned.

Done.

But I realized that the other derivatives entities have no such links in the derivatives pages. Should we add them in another PR ?

> want to open a follow-up issue in fMRIPrep

Should I do it?

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 2]
```diff
@@ -390,7 +390,7 @@ General fields:
          "For example, **T1**: `'sub-<label>/ses-<label>/anat/"
          "sub-01_T1w.nii.gz'`  "
          "**Surface**: `'/derivatives/surfaces/sub-<label>/ses-<label>/anat/"
-         "sub-01_desc-T1w_hemi-R_pial.surf.gii'` "
+         "sub-01_hemi-R_desc-T1w_pial.surf.gii'` "
```
Comment (Remi-Gau) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:393]: small change in this example to make it bids derivatives compliant

**Review by sappelhoff — APPROVED** [review]

**Review by effigies — APPROVED** [review]

**Comment by tsalo** [comment]
>But I realized that the other derivatives entities have no such links in the derivatives pages. Should we add them in another PR ?

I think that was just an oversight on my part. We should definitely link to the associated entity definition any time we reference an entity in the text.

**Review by tsalo — APPROVED** [review]
LGTM! I'll open an issue about the fMRIPrep derivatives.

EDIT: I've opened https://github.com/nipreps/fmriprep/issues/2620.

**Pull request merged**